1주차/4796/4796_python_후디.py

```diff
@@ -0,0 +1,15 @@
+case_list = []
+
+index = 0
+while True:
+    case = list(map(int, input().split(" ")))
+    if case[0] == case[1] == case[2] == 0:
+        break
+
+    l, p, v = case
```

Comment:
```suggestion
    l, p, v = list(map(int, input().split(" ")))
    if not l + p + v:
        break
```
아래서 case를 다시 할당하기보다 미리 할당하는 것도 괜찮아 보이네요.
추가로 l, p, v가 모두 양수이므로 이렇게 조건문을 작성할 수도 있어요!

Reply:
l, p, v 를 미리 할당하고, 합을 통해 조건식을 작성하니 확실히 더 보기 좋은 코드가 됐네요 😄